src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+syntax = "proto3";
+
+package aspire.v1;
+
+import "google/protobuf/timestamp.proto";
+
+////////////////////////////////////////////
+
+message ApplicationInformationRequest {
+}
+
+message ApplicationInformationResponse {
+    string application_name = 1;
+    string application_version = 2;
+}
+
+////////////////////////////////////////////
+
+message ResourceCommandRequest {
+    string command_type = 1;
```

Comment:
Could you please share the list of the command_type that are in use today?

Reply:
Currently there are none that I'm aware of. This is for future, so the question is whether the API shape is sufficient for the kinds of interactions we might want. Do you have some examples you might like to see?

Reply:
Thanks for clarifying! I don't have examples. Actually, what I had in mind was more that as we'll need to implement that same API in the Container Apps environment, we will need to understand to what we can map this. If this is a future API, maybe it's fine to leave it as non-implemented for now on our side.

Reply:
Each server side could decide which commands it wanted to expose, and the set could be empty. There's no requirement to use this, but it could be handy when you need it.